src/aleph/util.clj

```diff
@@ -0,0 +1,20 @@
+(ns aleph.util
+  (:require [manifold.deferred :as d]))
+
+(defn on-error
```

Comment:
I think it should be documented why you would use this over chain or on-realized.